apps/nextra/pages/en/build/create-aptos-dapp/templates/token-minting-dapp.mdx

```diff
@@ -15,118 +17,136 @@ The Token minting dapp template provides 3 pages:
 
 ## Quick Start
-- [Publish the Move module](#publish-the-move-module)
-- [Create a collection](#connect-a-wallet)
-- [Config the UI](#public-mint-page)
-- [Ready for Mainnet (optional)](#ready-for-mainnet)
+- [Create a token on chain](#publish-the-move-contract)
+- [Customizing the frontend](#customizing-the-front-end)
 - [Deploy to a live server](#deploy-to-a-live-server)
 
-## Public Mint Page
+## Setup
 
-First run the dapp locally with `npm run dev`. You would see the public mint page with some data placeholders for content and images. Some of the content can be configured in the `frontend/config.ts` file and some will be pulled from an Aptos full node once you configure your collection.
+You can start the app by running:
 
 ```bash 
 npm run dev
 ```
 
-### Configure an asset address
+However, for the app to work as intended, there are several setup steps you will have to do to:
 
-For the public mint page to fetch the asset, you will need to configure the asset address. First, assign the `fa_address` variable in the `frontend/config.ts` file with the asset address. If you have [created the asset](#create-a-fungible-asset-page) with the tool, you should be able to find it on the [My Assets Page](#my-assets-page).
+1. Publish the contract and create the token on-chain.
+2. Config your dapp to use the token
 
-### Modify static content
+## Publish the Move Contract
 
-Once the asset address has been configured, view the `frontend/config.ts` file to change any static content on the page. You can also modify the code itself as you wish.
+<Steps>
 
-#### How to add static images?
+### On the project root folder, use the below command to initiliaze an account to publish the contract with.
 
-The public mint page uses static images in the UI. Initially, the images are imported from the `frontend/assets/placeholder` folder. To use custom images, you should add the image you want to use to the `frontend/assets` folder (under any new folder you want to create) and then import the image as seen below in the `frontend/config.ts` file and add it under the section you want to have it.
-
-```ts filename="frontend/config.ts"
-import MyImage from "@/assets/<my-new-folder>/my-image.png";
+```bash 
+npm run move:init
 ```
 
-For example, to update an image in the "Our Team" section - add the image under the `frontend/assets/<my-new-folder>` folder, import the image as `import MyImage from "@/assets/<my-new-folder>/my-image.png";` and change the `img` property in the `ourTeam` section with `MyImage`.
+The CLI will prompt you to provide the private key for your wallet in order for the tool to publish the contract on your behalf. If you provide no input, it will generate one for you.
 
-## Create a Fungible Asset page
+This will create an account you will later publish the contract with, as well as setting the `VITE_MODULE_ADDRESS` in the `.env` file to set the contract object address.
```

Comment:
```suggestion
This will:
1. Create an account you will later publish the contract with
2. Set the `VITE_MODULE_ADDRESS` in the `.env` file to the contract object address.
```